### DISCUSSION TITLE
Setting RABBITMQ_BASE on Windows vs. setting

### DISCUSSION THREAD

**Discussion by rubin55** [rabbitmq/rabbitmq-website#1583] [General]
The documentation here: https://www.rabbitmq.com/relocate.html

Suggests that you can set `RABBITMQ_MNESIA_BASE` and `RABBITMQ_LOG_BASE` *instead* of setting the (windows-only) `RABBITMQ_BASE`:

*"Note: Windows only. This base directory contains sub-directories for the RabbitMQ server's database and log files. **Alternatively, set RABBITMQ_MNESIA_BASE and RABBITMQ_LOG_BASE individually**."*

However, if you don't set `RABBITMQ_BASE`, it will default to `APPDATA/Roaming/RabbitMQ` and specifically write the `db` directory and `enabled_plugins` file there, whilst honoring whatever you set  `RABBITMQ_MNESIA_BASE` and `RABBITMQ_LOG_BASE` to. 

This needs to be clarified better (i.e., indicate it not as "Alternatively" but as additionally).

**Comment by michaelklishin** [comment]
Both `RABBITMQ_MNESIA_BASE` and `RABBITMQ_LOG_BASE` are computed based on the value of `RABBITMQ_BASE`.

This docs do state the intended behavior: setting either `RABBITMQ_BASE` or the two more specific variables should be enough (assuming you want to override just data directory and log directory locations).

If that's not the case, we need a specific set of steps to reproduce.

Here are the relevant batch files that use those environment variables:

 * https://github.com/rabbitmq/rabbitmq-server/blob/a3344e2ea595c3e2d7e5dc8d73d79ef2f257c668/deps/rabbit_common/src/rabbit_env.erl#L617-L620
 * https://github.com/rabbitmq/rabbitmq-server/blob/v3.11.x/deps/rabbit/scripts/rabbitmq-defaults.bat
 * https://github.com/rabbitmq/rabbitmq-server/blob/v3.11.x/deps/rabbit/scripts/rabbitmq-env.bat
 * https://github.com/rabbitmq/rabbitmq-server/blob/v3.11.x/deps/rabbit/scripts/rabbitmq-server.bat
 * https://github.com/rabbitmq/rabbitmq-server/blob/v3.11.x/deps/rabbit/scripts/rabbitmq-service.bat

**Comment by michaelklishin** [comment]
I have pushed a doc update. It's not clear to me whether the docs have to be updated or this is a node startup behavior issue that has to be addressed.

The docs now say that overriding `RABBITMQ_BASE` is the optimal option for many.
